src/plugin/diskusage.cpp

```diff
@@ -39,6 +39,8 @@
 #include <QDir>
 #include <QDirIterator>
 
+Q_LOGGING_CATEGORY(diskUsage, "org.nemomobile.diskusage", QtWarningMsg)
```

Comment:
Looks like the other `Q_LOGGING_CATEGORY` names are prefixed with `org.sailfishos`, maybe this should be too?